typings/styled-components.d.ts

```diff
@@ -0,0 +1,162 @@
+  strong: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLElement>>, StyledProps<React.HTMLAttributes<HTMLElement>>>;
+  style: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLStyleElement>>, StyledProps<React.HTMLAttributes<HTMLStyleElement>>>;
+  sub: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLElement>>, StyledProps<React.HTMLAttributes<HTMLElement>>>;
+  summary: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLElement>>, StyledProps<React.HTMLAttributes<HTMLElement>>>;
+  sup: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLElement>>, StyledProps<React.HTMLAttributes<HTMLElement>>>;
+  table: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTableElement>>, StyledProps<React.HTMLAttributes<HTMLTableElement>>>;
+  tbody: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTableSectionElement>>, StyledProps<React.HTMLAttributes<HTMLTableSectionElement>>>;
+  td: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTableDataCellElement>>, StyledProps<React.HTMLAttributes<HTMLTableDataCellElement>>>;
+  textarea: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTextAreaElement>>, StyledProps<React.HTMLAttributes<HTMLTextAreaElement>>>;
+  tfoot: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTableSectionElement>>, StyledProps<React.HTMLAttributes<HTMLTableSectionElement>>>;
+  th: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTableHeaderCellElement>>, StyledProps<React.HTMLAttributes<HTMLTableHeaderCellElement>>>;
+  thead: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTableSectionElement>>, StyledProps<React.HTMLAttributes<HTMLTableSectionElement>>>;
+  time: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLElement>>, StyledProps<React.HTMLAttributes<HTMLElement>>>;
+  title: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTitleElement>>, StyledProps<React.HTMLAttributes<HTMLTitleElement>>>;
+  tr: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTableRowElement>>, StyledProps<React.HTMLAttributes<HTMLTableRowElement>>>;
+  track: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLTrackElement>>, StyledProps<React.HTMLAttributes<HTMLTrackElement>>>;
+  u: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLElement>>, StyledProps<React.HTMLAttributes<HTMLElement>>>;
+  ul: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLUListElement>>, StyledProps<React.HTMLAttributes<HTMLUListElement>>>;
+  "var": StyledFunction<React.ComponentClass<React.HTMLProps<HTMLElement>>, StyledProps<React.HTMLAttributes<HTMLElement>>>;
+  video: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLVideoElement>>, StyledProps<React.HTMLAttributes<HTMLVideoElement>>>;
+  wbr: StyledFunction<React.ComponentClass<React.HTMLProps<HTMLElement>>, StyledProps<React.HTMLAttributes<HTMLElement>>>;
+
+  // SVG
+  circle: StyledFunction<React.ComponentClass<React.SVGProps>, React.SVGAttributes<SVGCircleElement>>;
```

Comment:
Here in all SVG elements `StyledFunction` is used without `StyledProps`. Is that svg-specific where it cannot accept `theme` prop?